docker-compose.yaml

```diff
@@ -13,8 +13,8 @@ services:
 
   james:
     build:
-      context: local
-    image: james-local
+      context: .
+    image: james-server-and-cli
```

Comment:
Is this supposed to be for a single "custom" server only?

The original thought (before you kindly set up the gradle) is that it would be for running the "Local", "Advanced", and "Distributed" Servers (or whatever they will finally be called.

So this has morphed into something like "My Custom Server"? Is that the idea?

Reply:
Yes, a single server for now. 
But to run the advanced and all the variants you would have to change (in theory)
- the dependency line `implementation "org.apache.james:james-server-jpa-guice:${jamesVersion}"` 
- the configurations 

You could take this as a template and change the dependencies to use whatever James component you like. 
I'm going to focus on just one for now. I have lots of things to do and no time or interest for the others.

Reply:
Fair enough. Just wanted to make sure.

Also just to be sure: is it the "Basic" server you are working on? Or something else?